cmd/gapit/common.go

```diff
@@ -47,37 +47,6 @@ import (
 
 func (f CommandFilterFlags) commandFilter(ctx context.Context, client service.Service, p *path.Capture) (*path.CommandFilter, error) {
```

Comment:
This may disappear entirely?

Reply:
let's hold off on removing it entirely, since we are in discussion for other filtering.